config.yaml

```diff
@@ -1,6 +1,10 @@
 useFixtures: false
-
 elasticsearch:
+ host: 'https://search-talktofrank-elastic-alpha-rqcui3cnc7qs3i2ns5glqw4m7i.eu-west-2.es.amazonaws.com/'
+ drugResultCount: 3
+ suggestResultCount: 5
+ phraseResultCount: 3
+ showESResult: false
   host: 'https://search-talktofrank-elastic-alpha-rqcui3cnc7qs3i2ns5glqw4m7i.eu-west-2.es.amazonaws.com/'
```

Comment:
I think the lines above were removed in the latest branch I added, best to remove it as there will be two host values if the merge doesn't work out so well.